plugins/out_s3/s3.c

```diff
@@ -739,19 +765,13 @@ static int cb_s3_init(struct flb_output_instance *ins,
 
     tmp = flb_output_get_property("compression", ins);
```

Comment:
do you forget to delete this part? it is the same as line 640

Reply:
My bad, after merge with master. Code become messed up